src/applications/facility-locator/containers/FacilitiesMap.jsx

```diff
@@ -438,11 +437,8 @@ const FacilitiesMap = props => {
           <div id="search-result-emergency-care-info">
             <p className="search-result-emergency-care-subheader">
               <strong>Note:</strong> If you think your life or health is in
-              danger, call{' '}
-              <a aria-label="9 1 1" href="tel:911">
-                911
-              </a>{' '}
-              or go to the nearest emergency department right away.
+              danger, call <va-telephone contact="911" /> or go to the nearest
+              emergency department right away.
```

Comment:
I've tried to undo this change since it's out of scope for this PR, but each fixup commit ignores the changes and sees it as an "empty commit."